terraform/environments/mlra/modules/ALB/main.tf

```diff
@@ -0,0 +1,89 @@
+module "lb-access-logs-enabled" {
+  source = "github.com/ministryofjustice/modernisation-platform-terraform-loadbalancer"
+  providers = {
+    aws.bucket-replication = aws.bucket-replication
+  }
+
+  vpc_all                    = var.vpc_all
+  application_name           = var.application_name
+  public_subnets             = var.public_subnets
+  region                     = var.region
+  enable_deletion_protection = var.enable_deletion_protection
+  idle_timeout               = var.idle_timeout
+  force_destroy_bucket       = var.force_destroy_bucket
+  tags                       = var.tags
+  account_number             = var.account_number
+  loadbalancer_ingress_rules = local.loadbalancer_ingress_rules
+  loadbalancer_egress_rules  = local.loadbalancer_egress_rules
+}
+
+locals {
+  loadbalancer_ingress_rules = {
+    "cluster_ec2_lb_ingress" = {
+      description     = "Cluster EC2 loadbalancer ingress rule"
```

Comment:
Do we need 'cluster' in this description?

Reply:
cluster EC2 has been removed from the name and description